modules/rules/ipg.js

```diff
@@ -80,28 +86,55 @@ class IPG {
                 article.find(".entry-header, .page-navigation, em").remove();
 
                 const that = this;
-                article.find("h2, p, li, td, .card-header").each(function () {
-                    let text = $(this).text() + "\n";
-                    const nodeName = $(this).prop("nodeName").toLowerCase();
-                    if (nodeName === "h2" || nodeName === "div") {
-                        text = "\n**" + text + "**";
-                    }
-                    if (result.join("").length + text.length > that.maxLength) {
-                        msg.channel.sendMessage(result.join("").replace(/\n\s*\n\s*\n/g, '\n\n'));
-                        result = [];
-                    }
-                    result.push(text);
-                });
-                return msg.channel.sendMessage(result.join("").replace(/\n\s*\n\s*\n/g, '\n\n'));
+                if(parameters.length === 1){
+                    article.find("td").each((index,element)=>{
+                        result.push($(element).text()+"\n");
+                    });
+                    article.find("h2, p, li, .card-header").each((index,element)=>{
+                        let text = $(element).text()+"\n";
+                        const nodeName = $(element).prop("nodeName").toLowerCase();
+                        if(nodeName==="h2" || nodeName==="div"){
+                            text = "\n**"+text+"**";
+                        }
+                        result.push(text);
+                        if(result.join("").length>that.maxPreview){
+                            return false;
+                        }
+                    });
+                }else{
+                    const topics = parameters.slice(1,parameters.length);
```

Comment:
Yeah could be to long and I don't know if it will be used anyway. A `full` parameter could be more useful bit I think that's to much text.